### ISSUE TITLE
standard: plugings ?

### ISSUE THREAD

**Issue by NorthDecoder** [briancavalier/tomato#1]
In file `package.json`

**Should this object:**
```javascript
  "standard": {
    "parser": "babel-eslint",
    "plugings": [
      "flowtype"
    ]
}
```
**be:**
```javascript
  "standard": {
    "parser": "babel-eslint",
    "plugins": [
      "flowtype"
    ]
}
```
where `plugings` is changed to `plugins` ?

**Comment by briancavalier** [comment]
Oops! Thanks. Clearly these tomatoes aren’t anywhere near ripe 😄. I’ll fix it soon!

**Comment by briancavalier** [comment]
Fixed by 215dc83

Thanks again. Cheers!
